supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift-container-storage.adoc

```diff
@@ -0,0 +1,147 @@
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[container-storage-interface]]
+==== Container Storage Interface (noun)
+*Description*: The Container Storage Interface (CSI) is a standard for exposing arbitrary block and file storage systems to containerized workloads on Container Orchestration Systems like Kubernetes, and in particular Red Hat OpenShift Container Platform. This allows OpenShift Container Platform to consume storage from third-party storage providers that implement the CSI interface as persistent storage.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[multicloud-object-gateway]]
+==== Multicloud Object Gateway (noun)
+*Description*: Multicloud Object Gateway (MCG) is a lightweight object storage service for OpenShift Container Platform you can use to start with a small storage service and then scale according to your requirements on-premises, in multiple clusters, and with cloud-native storage.
+
+Multicloud Object Gateway (MCG) is a lightweight object storage service for OpenShift Container Platform that you can use to start with a small storage service and then scale according to your requirements on-premise, in multiple clusters, and with cloud-native storage.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[namespace-bucket]]
+==== namespace bucket (noun)
+*Description*: Namespace buckets enable connecting data repositories on different providers together so that the interaction with data is possible through a single unified view. Data written to namespace buckets are kept plain and not deduped, compressed, or encrypted. The data can be consumed directly from the repository.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[namespace-store]]
+==== namespace store (noun)
+*Description*: A type of a resource for Multicloud Object Gateway that is used to store plain data and is used by namespace buckets. The supported stores are S3, Azure, and S3 compatible.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[object-bucket]]
+==== object bucket (noun)
+*Description*: A container for storing objects.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[object-bucket-claim]]
+==== object bucket claim (noun)
+*Description*: Creates a new bucket and an application account in Multicloud Object Gateway or RADOS Gateway (RGW) with permissions to the bucket, including a new access key and secret access key. An object bucket claim can be used to request an S3 compatible bucket for the workloads.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[persistent-volume]]
+==== persistent volume (noun)
+*Description*: A persistent volume (PV) is a piece of storage in the cluster that has been provisioned by an administrator or dynamically provisioned using storage classes. 
```

Comment:
Passive > active suggestion:
Also, because of the problematic 'using' here, there are two possible translations so choose whichever is accurate: 

This suggestion assumes that the storage classes do the action of provisioning:
`A persistent volume (PV) is a piece of storage in the cluster that an administrator provisions or that storage classes dynamically provision.`

This suggestion assumes that the admin uses the storage classes to do the action of provisioning:
`A persistent volume (PV) is a piece of storage in the cluster that an administrator provisions or uses storage classes to dynamically provision.`

IBM: 
**using** noun
To avoid ambiguity, replace this gerund with either "by using" or "that uses".
To refer to something "using" one thing or another is often ambiguous. For example, the sentence "Back up the server using ABC" might mean "Back up the server by using ABC" or "Back up the server that uses ABC". This advice also applies to gerunds other than "using". For more information, see the style and grammar tips in the Global audiences topic in IBM Style and the section about "Clarity" in "Developing Quality Technical Information".

Reply:
Done. (second option)